src/apiGateway.ts

```diff
@@ -37,12 +37,14 @@ export function httpRequestToEvent(request: Request): APIGatewayProxyEventV2 {
     const bodyString = request.method === 'GET' ? '' : request.body.toString('utf8');
     const shouldSendBase64 = request.method === 'GET' ? false : bodyString.includes('Content-Disposition: form-data');
 
+    const cookies = request.headers.cookie ? request.headers.cookie.split('; ') : [];
```

Comment:
I'm afraid this might break if a cookie contains the `;` character, could you add a test to cover that scenario?

Reply:
cookies, like most things HTTP, need to be URL-encoded.

This is a cookie with `;`

<img width="802" alt="image" src="https://github.com/brefphp/local-api-gateway/assets/9533181/85261db1-fa1c-45b7-adb0-ef8cead63c6d">

which is what makes it safe to do the split on `;`.

---
I'm trying to understand the tests to see if I can add one

Reply:
🤦 of course sorry, that makes sense!